api/bucketsd/service.go

```diff
@@ -2332,12 +2333,13 @@ func (s *Service) Archive(ctx context.Context, req *pb.ArchiveRequest) (*pb.Arch
 		return nil, fmt.Errorf("parsing cid path: %s", err)
 	}
 
-	createNewFFS := func() error {
-		id, token, err := s.PowergateClient.FFS.Create(ctx)
+	createNewUser := func() error {
+		ctxAdmin := context.WithValue(ctx, pow.AdminKey, s.PowergateAdminToken)
```

Comment:
This feels a little spaghetti to have to have the Powergate admin token set on `Service` and then use it here to build a context...

One day I plan on making the Powergate client itself statefuly manage the token and internally create/use the grpc metadata so the client can just be configured once and passed around as needed.